app/src/main/kotlin/no/nav/k9punsj/db/datamodell/FagsakYtelseType.kt

```diff
@@ -16,7 +16,7 @@ enum class FagsakYtelseType(val kode: String, val navn: String, val infotrygdBeh
             if (type != null) {
                 return type
             } else {
-                throw IllegalStateException("Fant ingen FagsakYtelseType med koden $kode")
+                return UKJENT
```

Comment:
Er det ikke bedre å lage en FagsakYtelseType for OLP her? For det er OLP det feiler på, ikke sant?
Her risikerer vi lagre mye rusk med av typen UKJENT?

Reply:
Eventuelt kaste en custom exception og håndtere den i listener. Slik at vi kan ignorere de meldingene som vi kke støtter.

Reply:
Då er det vel en bedre løsning att ikke lagre journalposter av typen UKJENT, sitter och ser på det her nå... Må justere detsamme for innsendingstype, for per nå så kraschar consumern så fort vi får en "poison pill" på kafka och vi får ingen varsel på att den er nere.

Reply:
Jeg er usikker på hviket journalposter vi får av typen UKJENT, det burde vi få sjekket ut.
Det er ikke bra at vi ikke har noe varsling på dette.

Reply:
Fortsetter diskussionen på slack, en del vi trenger avklaringer på her.